tests/main.test.ts

```diff
@@ -110,22 +110,53 @@ describe("User start/stop", () => {
 
     context.adapters = createAdapters(getSupabase(), context as unknown as Context);
 
-    const output = await userStartStop(context as unknown as Context);
+    const logReturn: LogReturn = {
+      logMessage: {
+        diff: "```diff\n! You are not assigned to this task\n```",
+        level: "error",
+        raw: "You are not assigned to this task",
+        type: "error",
+      },
+      metadata: {
+        caller: "error",
+        issueNumber: 2,
+        user: "ubiquity",
+      },
+    };
 
-    expect(output).toEqual({ output: "You are not assigned to this task" });
+    try {
+      expect(await userStartStop(context as unknown as Context)).toThrow(logReturn);
+    } catch (error) {
+      expect(error).toEqual(logReturn);
+    }
```

Comment:
because you awaited the function inside expect, it will return a value so `toThrow` is meaningless, but it will still work because you catch the error.

I recommend:
```suggestion
    await expect(userStartStop(context as unknown as Context)).rejects.toThrow(logReturn)
```

Reply:
Thank you, I was scratching my head as to why it wasn't working like this. I was getting a ts error putting `await` before expect but perhaps it was because I had awaited the function too.

Reply:
Unsure what the issue is but your suggestion is not working as expected.

without the try-catch it tells me that the function did not throw but you see in the screenshot below inside the try-catch it's displaying our `LogReturn` that we throw... Seen anything like this before?

![image](https://github.com/user-attachments/assets/795c60ce-7214-46db-9525-7665f36a63e9)

Reply:
I'm not sure but it's in the [docs](https://jestjs.io/docs/expect#rejects) so it should work

Reply:
the issue seemed to be that I wasn't throwing an `Error` so it wasn't registering